test/metabase/test/data/exasol.clj

```diff
@@ -38,10 +38,13 @@
 
 (defmethod tx/has-questionable-timezone-support? :exasol [_] true)
 
+(defonce ^:private number-column-type
+  "DECIMAL(36,0)")
+
 (doseq [[base-type sql-type] {:type/Text                   "VARCHAR(4000)"
-                              :type/BigInteger             "DECIMAL(36,0)"
-                              :type/Integer                "DECIMAL(18,0)"
-                              :type/Decimal                "DECIMAL(18,0)"
+                              :type/BigInteger             number-column-type
```

Comment:
Does it also work with decimalslike decimal(10,2)

Reply:
This file is only used as configuration for tests containing DB specific details, e.g. which DB data type to use for Metabase's internal data types. If there was a problem, the integration tests would fail.